### ISSUE TITLE
Better error state if Kodi host is invalid

### ISSUE THREAD

**Issue by hash-bang** [CMP2804M-Group3/kodi-controller#86]
When trying to connect to a non-responsive or invalid host the module always assumes that a valid `playerId` is present, causing a crash upstream:

Running:

```javascript
var KodiController = require('kodi-controller');
var kodi = new KodiController('nope.nope.nope', '8080');
```

Throws:

```
./node_modules/kodi-controller/src/Controller.js:116
            callback(err, data[0].playerid);
                                  ^

TypeError: Cannot read property 'playerid' of undefined
    at /home/mc/Dropbox/Projects/Node/kodi-controller-cli/node_modules/kodi-controller/src/Controller.js:116:35
    at Request.<anonymous> (/home/mc/Dropbox/Projects/Node/kodi-controller-cli/node_modules/kodi-controller/src/Controller.js:91:32)
    at Request.emit (events.js:228:7)
    at IncomingMessage.<anonymous> (/home/mc/Dropbox/Projects/Node/kodi-controller-cli/node_modules/request/request.js:1076:12)
    at Object.onceWrapper (events.js:312:28)
    at IncomingMessage.emit (events.js:228:7)
    at endReadableNT (_stream_readable.js:1185:12)
    at processTicksAndRejections (internal/process/task_queues.js:81:21)
```

It is possible to work around this by enclosing the module in a Domain but its an ugly fix:

```javascript
return new Promise((resolve, reject) => {
	var domain = require('domain').create();
	domain.on('error', e => reject(`Unable to connect to Kodi player @ "${program.host}:${program.port}"`));
	domain.run(()=> kodi.getActivePlayerID())
})
	.then(()=> /* Sunshine and lollipops */)
	.catch(()=> /* Doom and despair */)
```

**Comment by Pittsy24** [comment]
Just had a quick look, adding a return after the callback(err) in the get player ID function should fix it that way the function is not trying to callback with a player Id it doesn't have. 

@benlincoln could you have a look into this?

**Comment by hash-bang** [comment]
Thanks for the fix.

Would you mind bumping the version and republishing? I think this may also fix #85.
